api/service/legacysync/helpers.go

```diff
@@ -0,0 +1,99 @@
+package legacysync
+
+import (
+	"sync"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common/math"
+	"github.com/harmony-one/harmony/api/service/legacysync/downloader"
+	"github.com/harmony-one/harmony/internal/utils"
+	"github.com/harmony-one/harmony/p2p"
+	"github.com/pkg/errors"
+)
+
+// getMaxPeerHeight gets the maximum blockchain heights from peers
+func getMaxPeerHeight(syncConfig *SyncConfig) uint64 {
```

Comment:
Just to confirm. the code in this file helpers.go is directly copy from the original code without functional change, right?

Reply:
yes, copy paste